index.html

```diff
@@ -12,6 +12,28 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <nav>
+            <img class="logo-img" src="img/karma-logo.svg" alt="My logo">
+            <ul class="ul-header">
+                <li> <a>login</a> </li>
+                <li> <a>Store</a> </li>
+                <li> <a>Home</a> </li>
+                <li> <a>Blog</a> </li>
+                <li> <a>Help</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero-sec">
+            <img class="hero-img" src="img/first-background.jpg" alt="myPic" >
```

Comment:
Make sure you include a meaningful description of your image in the `alt` attribute. Users of assistive technology such as screen readers rely on these descriptions to be able to *see* your images!